(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> _and() takes at least 3 arguments (3 given)"
p3
sS'layer'
p4
S'/Users/EmmyB/Desktop/web2py/applications/cs183project/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x10fd49230, file "/Users/EmmyB/Desktop/web2py/applications/cs183project/controllers/default.py", line 10>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/Users/EmmyB/Desktop/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/Users/EmmyB/Desktop/web2py/applications/cs183project/controllers/default.py", line 171, in <module>\n  File "/Users/EmmyB/Desktop/web2py/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/Users/EmmyB/Desktop/web2py/applications/cs183project/controllers/default.py", line 153, in search\n    rows = db(q).select(db.property.ALL)\n  File "/Users/EmmyB/Desktop/web2py/gluon/packages/dal/pydal/objects.py", line 2250, in select\n    return adapter.select(self.query, fields, attributes)\n  File "/Users/EmmyB/Desktop/web2py/gluon/packages/dal/pydal/adapters/sqlite.py", line 82, in select\n    return super(SQLite, self).select(query, fields, attributes)\n  File "/Users/EmmyB/Desktop/web2py/gluon/packages/dal/pydal/adapters/base.py", line 757, in select\n    colnames, sql = self._select_wcols(query, fields, **attributes)\n  File "/Users/EmmyB/Desktop/web2py/gluon/packages/dal/pydal/adapters/base.py", line 672, in _select_wcols\n    query = self.expand(query, query_env=query_env)\n  File "/Users/EmmyB/Desktop/web2py/gluon/packages/dal/pydal/adapters/base.py", line 436, in _expand\n    rv = op(first, **optional_args)\nTypeError: _and() takes at least 3 arguments (3 given)\n'
p11
s.